docs/FAQ.md

```diff
@@ -44,3 +44,16 @@ $ export QUILT_MINIMIZE_STDOUT=true
 1. Go to CloudFormation > Stacks > YourQuiltStack > Outputs
 1. Copy the row labeled TemplateBuildMetadata
 1. "git_revision" is your template version
+
+## Hashing takes a long time. Can I speed it up?
+
+Yes. Follow these steps:
+
+1. Run your compute in the same region as your S3 bucket (as opposed to
+a local machine or foreign region)—I/O is much faster.
+
+1. Use a larger instance with more vCPUs.
+
+1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](API%20Reference/cli.md#quilt_transfer_max_concurrency)
+above its default of 10 to better
```

Comment:
I'd rather not repeat here what value is default.

Reply:
True